tests/unit/show_tuner_decisions.c

```diff
@@ -30,7 +30,7 @@ int main(int argc, const char **argv)
 				return 1;
 			}
 
-			for (size_t nmibytes = 1; nmibytes <= 32 * 1024; nmibytes <<= 1) {
+			for (size_t nmibytes = 1; nmibytes <= 32 * 1024 * 1024; nmibytes <<= 1) {
```

Comment:
This variable should be named `nkibytes` now.

Reply:
Fixed this.